Automattic/wp-calypso
# Add domain upsell nudge for plan owners with domain

**Changes proposed in this Pull Request**

Adds the domain upsell nudge, discussed here: p99Zz8-K5-p2

Depends on D32926-code

**Testing instructions**

- Verify the nudge only appears to sites with a plan and at least one domain, if there are no other notices/nudges
- Make sure it displays the correct data (hack the backend to return custom price and sale data and verify the nudge displays correct information)
